README.md

```diff
@@ -10,12 +10,328 @@ A core widget library for Dojo 2.
 
 For more background on Widgets for Dojo 2, there is a document describing the [widgeting system](https://github.com/dojo/meta/blob/master/documents/Widget-System.md).
 
+- [Usage](#usage)
+- [Features](#features)
+    - [Base Widget](#base-widget)
+    	- [Simple Widgets](#simple-widgets)
+    	- [d](#d)
+    	- [Widgets with Children](#widgets-with-children)
+    - [Extending Base Widget](#extending-base-widget)
+    - [Projector](#projector)
+    - [Dojo Widget Components](#dojo-widget-components)
+- [How do I contribute?](#how-do-i-contribute)
```

Comment:
Extreme nitpick... I'd say that everything should either be uppercase or lowercase after the first word except proper nouns. I tend to favour the latter, though you favour the former in most of the titles, except for a few. :)